Zapic/Controllers/ZScriptMessageHandler.m

```diff
@@ -165,11 +165,12 @@ - (void)handleShowShare:(nonnull NSDictionary *)data {
 
 - (void)handleLogin:(nonnull NSDictionary *)data {
     NSDictionary *msg = data[@"payload"];
-    NSString *userId = msg[@"userId"];
+    NSString *identifier = msg[@"id"];
```

Comment:
@dsarfati We already have a function that gets the current SDK version that can be used here. Though, I did have a pull request merged on the JS side that changed userId to id. I will make a quick PR to fix that.

Reply:
I think we should ask @seniorquico what we want to do here. I would definitely do a PR to switch it back to userId for now until we get this figured out

Reply:
Just submitted a PR to include userId again on the JS side.

Reply:
@scottcinatl Let's change this to use "userId" in the SDK and merge it in. I dont think its worth going through the pain of changing both JS & SDKs to change something that neither developers or users will notice. We will still call the property identifier in the objc code so that will change, but without changing our internal models. As soon as you do this simple change, ill merge it in